WebRoot/js/zimbraMail/core/ZmOperation.js

```diff
@@ -185,7 +185,7 @@ function() {
 		AjxCallback.simpleClosure(function(parent) {
 			ZmOperation.addDeferredMenu(ZmOperation.addSearchMenu, parent, true);
 		}));
-	ZmOperation.registerOp(ZmId.OP_SEND, {textKey:"send", tooltipKey:"sendTooltip", image:"Send", shortcut:ZmKeyMap.SEND});
+	ZmOperation.registerOp(ZmId.OP_SEND, {textKey:"send", tooltipKey:"sendTooltip", image:"Send", shortcut:ZmKeyMap.SEND, showImageInToolbar: true, showTextInToolbar: true});
```

Comment:
Modified the 'overrides' object, here.
https://github.com/Zimbra/zm-web-client/commit/b2c6653e4ee460ed03a11702ea11f17725bac5ca